databricks-sdk-java/src/main/java/com/databricks/sdk/core/ApiClient.java

```diff
@@ -202,6 +267,8 @@ private <I> Request prepareBaseRequest(String method, String path, I in)
     } else if (InputStream.class.isAssignableFrom(in.getClass())) {
       InputStream body = (InputStream) in;
       return new Request(method, path, body);
+    } else if (in instanceof String) {
```

Comment:
Suggestion: leave a comment that this assumes that the request is already serialized.